### ISSUE TITLE
Ingest Gs0154189

### ISSUE THREAD

**Issue by aclum** [microbiomedata/issues#910]
We have 20 metagenomes + JGI metabolomics 

MSV000094088
blocked on environmental terms in GOLD.

**Comment by aclum** [comment]
made a gold ticket

**Comment by aclum** [comment]
terms have been updated in GOLD.

**Comment by aclum** [comment]
Not a priority for the renewal.
